gradle/libs.versions.toml

```
@@ -10,7 +10,7 @@ hamcrest = "2.2"
 nekohtml = "1.9.22"
 htmlunit = "3.4.0"
 lang3 = "3.11"
-io = "2.11.0"
+commons_io = "2.13.0"
```

```suggestion
commons_io = "2.15.0"
```
Can we update to most recent version?